pkg/splunk/client/s3client.go

```diff
@@ -6,15 +6,17 @@ import (
 )
 
 //GetS3Client gets the required S3Client based on the provider
-type GetS3Client func(string /* region */, string /* bucket */, string, /* AWS access key ID */
-	string /* AWS secret access key */, string /* Prefix */, string /* StartAfter */) S3Client
+type GetS3Client func(string /* bucket */, string, /* AWS access key ID */
```

Comment:
nit: `.., string /* AWS access key ID */, string....`(sorry it was my bad :) )

Reply:
I got you.  Fixed.